src/system/sys_profiler.cc

```diff
@@ -350,7 +350,43 @@ std::string common_prefix( std::string a, std::string b ) {
     return std::string( a.begin(), std::mismatch( a.begin(), a.end(), b.begin() ).first ) ;
 }
 
+void Profiler::print_timer_path (int timer_index) {
```

Comment:
Is the change in profiler necessary for this pull? It should rather be part of separate pull.
However, it looks like forgotten debugging hack.